aws_cloudtrail_rules/aws_resource_made_public.py

```diff
@@ -23,6 +23,10 @@ def rule(event):
 
     # S3
     if event['eventName'] == 'PutBucketPolicy':
+        # Don't alert if access is denied
```

Comment:
We can actually put this block under the `if not parameters` check since it could apply to any resource type!

It can also be simplified to:

```python
if event.get('errorCode') == 'Access Denied':
  return False
```

Reply:
Modified as suggested.